mne_bids/tests/test_read.py

```diff
@@ -317,6 +317,25 @@ def test_handle_coords_reading():
     with pytest.raises(RuntimeError, match='Channels do not correspond'):
         raw_test = read_raw_bids(bids_fname, bids_root)
 
+    # make sure montage is not set if there are coordinates w/ 'n/a'
+    # but not set in raw.info['bads']
+    raw.info['bads'] = []
+    write_raw_bids(raw, bids_basename, bids_root, overwrite=True)
+    electrodes_dict = _from_tsv(electrodes_fname)
+    for axis in ['x', 'y', 'z']:
+        electrodes_dict[axis][0] = 'n/a'
+        electrodes_dict[axis][3] = 'n/a'
+    _to_tsv(electrodes_dict, electrodes_fname)
+    channels_fname = _find_matching_sidecar(bids_fname, bids_root,
+                                            "channels.tsv",
+                                            allow_fail=True)
+    channels_dict = _from_tsv(channels_fname)
+    channels_dict['status'][0] = 'good'
+    channels_dict['status'][3] = 'good'
```

Comment:
I don't get what you are doing here.

Reply:
The idea is to set electrode coordinates to `n/a`, but channel status to either `good`, or `bad`, which should result in the montage not being set, or being set respectively.

Reply:
but how do you check that? I see that in both cases you have `raw_test.info['dig'] is not None`

Reply:
Sorry, yeah in both cases, according to the conversation, raw_test.info['dig'] is present. The only difference is that the "good" channels are now set to bad if they are not present in electrodes.tsv. I updated the unit test to reflect it.